src/vs/workbench/contrib/notebook/browser/services/notebookExecutionServiceImpl.ts

```diff
@@ -105,6 +105,20 @@ export class NotebookExecutionService implements INotebookExecutionService, IDis
 		this.cancelNotebookCellHandles(notebook, Array.from(cells, cell => cell.handle));
 	}
 
+	private readonly cellExecutionParticipants = new Set<ICellExecutionParticipant>;
+
+	registerExecutionParticipant(participant: ICellExecutionParticipant) {
+		this.cellExecutionParticipants.add(participant);
+		return toDisposable(() => this.cellExecutionParticipants.delete(participant));
+	}
+
+	async runExecutionParticipants(executions: INotebookCellExecution[]): Promise<void> {
```

Comment:
This should be private unless there's a reason to expose it publicly

Reply:
Fixed by: [183164](https://github.com/microsoft/vscode/pull/183164)